From e660a950c12e1ea28d6513c373c24cc6a60e6cf5 Mon Sep 17 00:00:00 2001
From: Volodymyr Babchuk <volodymyr_babchuk@epam.com>
Date: Sun, 4 Apr 2021 01:05:42 +0300
Subject: [PATCH 11/13] rvgc: add plane_state parameter to rvgc_pipe_enable()

Backported patch ebe81ac24a ("drm/simple-kms-helper: Plumb plane state to the
enable hook") added new parameter to .enable callback. We need to add the same
parameter to the rvgc_pipe_enable() function.

Signed-off-by: Volodymyr Babchuk <volodymyr_babchuk@epam.com>
---
 drivers/gpu/drm/rcar-rvgc/rcar_rvgc_pipe.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/rcar-rvgc/rcar_rvgc_pipe.c b/drivers/gpu/drm/rcar-rvgc/rcar_rvgc_pipe.c
index 88c9cf0714e5..2186146d838c 100644
--- a/drivers/gpu/drm/rcar-rvgc/rcar_rvgc_pipe.c
+++ b/drivers/gpu/drm/rcar-rvgc/rcar_rvgc_pipe.c
@@ -156,7 +156,8 @@ rvgc_connector_create(struct rcar_rvgc_pipe *rvgc_pipe)
 }
 
 static void rvgc_pipe_enable(struct drm_simple_display_pipe *pipe,
-			struct drm_crtc_state *crtc_state)
+			     struct drm_crtc_state *crtc_state,
+			     struct drm_plane_state *plane_state)
 {
 	struct device *ddev = pipe->plane.dev->dev;
 	struct rcar_rvgc_pipe *rvgc_pipe = container_of(pipe, struct rcar_rvgc_pipe, drm_simple_pipe);
-- 
2.31.0

